build/images/Dockerfile.build.windows

```diff
@@ -52,3 +52,10 @@ COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/libeay32.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/ssleay32.dll /Windows/System32/
 COPY --from=windows-ovs /openvswitch/usr/bin /k/antrea/bin/
+COPY --from=windows-ovs /openvswitch/usr/sbin /k/antrea/bin/
+
+RUN mkdir C:\openvswitch
+COPY --from=windows-ovs /openvswitch /openvswitch/
+
+RUN Remove-Item "C:\openvswitch\usr\bin" -Recurse
+RUN Remove-Item "C:\openvswitch\usr\sbin" -Recurse
```

Comment:
This will not really reduce the image size given they are added to the above layer.

Since we are using an unified image now, I wonder if we can now keep the ovs bins in `C:\openvswitch\`?

Reply:
Does the antrea-windows-containerd.yml work well if we keep OVS bins in C:\openvswitch\ instead of C:\k\antrea\bin\ ？

Reply:
Yes, I'm thinking the same as @XinShuYang. Adding OVS's bin dirs to PATH should be harmless for antrea-windows-containerd.

Reply:
I believe YES, because OVS bins weren't present in the image before and antrea-windows-containerd.yml worked well as it does not use OVS bins from image. In case of antrea-windows-containerd.yml OVS is already running as windows service on host and it uses that from host.

Reply:
done.